randovania/games/am2r/logic_database/header.json

```diff
@@ -904,7 +904,7 @@
             "MissileLessMetroids": {
                 "long_name": "Missile-Less Metroid Fights",
                 "description": "If this trick is enabled, the player may be expected to kill Metroids with only Charge Beam.",
-                "require_documentation_above": 0,
+                "require_documentation_above": 1,
```

Comment:
wonder if this should be 0 instead? alphas all use one template, adding one video there wouldnt hurt.

Reply:
Were dodging vs non-dodging not split?
honestly if we just reuse the same video can just stay 0 then.